docs/guide/input-validation.md

```diff
@@ -372,6 +373,14 @@ class MyForm extends Model
 }
 ```
 
+> Note: you can use [[yii\validators\InlineValidator::addError()]] for adding errors instead. That way the error
+> message can be formatted using [[yii\i18n\I18N::format()]] right away. Use `{attribute}` and `{value}` in the error
```

Comment:
Further explanation on this: in the guide the context around is the root namespace so it is like in PHP itself, when you are in a namespace you need `\` to get to the root. In root namespace you do not need it.